controller/volume_controller.go

```diff
@@ -549,9 +549,12 @@ func (vc *VolumeController) ReconcileEngineReplicaState(v *longhorn.Volume, es m
 	if e.Status.CurrentState != longhorn.InstanceStateRunning {
 		// If a replica failed at attaching stage before engine become running,
 		// there is no record in e.Status.ReplicaModeMap
+		engineInstanceCreationCondition := types.GetCondition(e.Status.Conditions, longhorn.InstanceConditionTypeInstanceCreation)
+		isNoAvailableBackend := strings.Contains(engineInstanceCreationCondition.Message, fmt.Sprintf("exit status %v", int(syscall.ENODATA)))
```

Comment:
Where to cause ` fmt.Sprintf("exit status %v", int(syscall.ENODATA)))`? Didn't see any code from engine creation (controller command)

Reply:
Set the exit code ENODATA here.
https://github.com/longhorn/longhorn-engine/pull/780/files#diff-33f40fb2b9aa0391a4eec1808d3248f24494e63100dd5defe18370f3f2236390R172